File: examples/advanced-crud/src/main/java/org/example/advancedcrud/app/repository/MovieActorRepository.java
package org.example.advancedcrud.app.repository;

import org.example.advancedcrud.app.entity.MovieActor;
import org.example.advancedcrud.app.entity.MovieActorId;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

public interface MovieActorRepository extends JpaRepository<MovieActor, MovieActorId>,
        JpaSpecificationExecutor<MovieActor> {
}

File: examples/basic-crud-kotlin/src/main/kotlin/org/example/basiccrudkotlin/app/repository/MovieActorRepository.kt
package org.example.basiccrudkotlin.app.repository;

import org.example.basiccrudkotlin.app.entity.MovieActor
import org.example.basiccrudkotlin.app.entity.MovieActorId
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor
import org.springframework.data.rest.core.annotation.RepositoryRestResource


@RepositoryRestResource(path = "movieActors")
interface MovieActorRepository : JpaRepository<MovieActor, MovieActorId>, JpaSpecificationExecutor<MovieActor> {
}